.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: build
+on:
+  release:
+    types: [created]
+  push:
+    branches:
+      - "**"
+    paths:
+      - "header-gen/**"
+      - "**/build.yml"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./header-gen
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "20.x"
+          registry-url: "https://npm.pkg.github.com"
+          cache: npm
+          cache-dependency-path: header-gen/package-lock.json
+      - run: npm ci
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: microbit-foundation/npm-package-versioner-action@v1
+        with:
+          working-directory: ./header-gen
```

Comment:
Assuming the workflow will be to keep updating the extension to `pxt bump`, and to manually create a GH release from an existing tag when need to push this package, then that means the package will follow the same version as the pxt extension.

I think this is perfectly fine, just wanted to double check this is the intention and the proposed workflow.

Reply:
I think this makes sense.